src/autofill-utils.js

```diff
@@ -392,6 +392,20 @@ const wasAutofilledByChrome = (input) => {
     }
 }
 
+/**
+ *
+ * @param {Function} callback
+ * @param {number} timeout
+ * @returns {Function}
+ */
+function debounce (callback, timeout = 150) {
+    let timer
+    return (...args) => {
+        clearTimeout(timer)
+        timer = setTimeout(() => callback.apply(this, args), timeout)
```

Comment:
Have you tried using `requestIdleCallback` instead of `setTimeout`? It's usually be better suited for debouncing because it schedules the action right when the browser is ready vs forcing priorities.